### DISCUSSION TITLE
Initial model recognition error

### DISCUSSION THREAD

**Discussion by lixinyu1011** [deepmodeling/reacnetgenerator#1344] [Q&A]
Hello Professor, I successfully installed the software you developed, but I encountered some problems.
I constructed a polyester system composed of succinic acid and butanediol. One polymer chain in the box consists of 18 succinic acids and 18 butanediol. After the lammps operation, the generated is processed by the command （reacnetgenerator --type lammpsdumpfile -i dump.reaxc -a C H O）. But the substance identified in the result is incorrect. What is the reason?
I look forward to your answer.

The initial model is as follows：
![image](https://user-images.githubusercontent.com/61094742/138877375-91da44ee-795e-4346-a998-8f09748f10a1.png)
The model identified after processing is as follows：
![image](https://user-images.githubusercontent.com/61094742/138877549-5a8bec80-ba93-42d0-80c9-dab357ac6160.png)

**Comment by njzjz** [comment]
Are you sure your type is C H O but not something like C O H?
